packages/openapi3-parser/lib/parser/oas/parseOauthFlowsObject.js

```diff
@@ -72,7 +72,8 @@ function parseOauthFlowsObject(context, object) {
       authScheme.push(new namespace.elements.Member('grantType', grantTypes[member.key.toValue()]));
       authScheme.push(member.value.getMember('scopes'));
 
-      R.filter(R.is(namespace.elements.Transition), member.value).forEach((item) => {
+      const isTransitionElement = R.is(namespace.elements.Transition);
+      member.value.filter(isTransitionElement).forEach((item) => {
```

Comment:
This one was a little tricky to grapple with. `member.value` here is an `ObjectElement`. An object element is a series of key/value pairs called members.

ObjectElement's [forEach has the signature of `(value, key, item)`](https://github.com/refractproject/minim/blob/master/lib/primitives/ObjectElement.js#L197). Thus when `filter` returns an ObjectElement, the behaviour changed. I have switched to `member.value.filter` which is equivalent to the prior behaviour as it would call `forEach` on ArraySlice which iterates over each member element and doesn't have the forEach semantics of an object.